src/events/cd-event-list.vue

```diff
@@ -8,11 +8,18 @@
         <div>
           <span class="cd-event-list__no-events-content" v-html="$t('There are no upcoming events planned for this Dojo.')"></span>
           <span class="cd-event-list__no-events-content" v-html="$t(`${dojo.private ? 'Please email {email} if you have any questions.' : 'Please join this Dojo for updates or email {email}' }`, { email: '<a href=\'mailto:' + dojo.email + '\'>' + dojo.email + '</a>' })"></span>
-          <button @click="joinTheDojo()" v-if="currentUser && !dojo.private && usersDojos.length === 0" class="cd-event-list__no-events-join-button">{{ $t('Join Dojo') }}</button>
         </div>
+        <button @click="joinTheDojo()" v-if="!dojo.private && !isDojoMember" class="cd-event-list__no-events-join-button">{{ $t('Join Dojo') }}</button>
```

Comment:
is there a reason its not in the div anymore?

Reply:
Yep : block element on new line so that the button isn't "chaining" with the sentence on responsive